src/Models/Menu.php

```diff
@@ -90,4 +99,49 @@ public function buildTree($elements, $parentId = 0)
 
         return $branch;
     }
+
+    /**
+     * @param string $type
+     * @return array
+     */
+    public function getRoutes($type = 'admin'): array
```

Comment:
It looks like everything related to routes can be moved to separate service class